x-pack/platform/plugins/shared/security/server/session_management/session_index.ts

```diff
@@ -786,8 +785,8 @@ export class SessionIndex {
         index: this.aliasName,
         document: sessionValueToStore,
         refresh: false,
-        require_alias: true,
-      } as CreateRequest,
+        querystring: { require_alias: true },
```

Comment:
__question:__ Hmm, I believe this isn't related to this PR, but it looks like we don’t have a definition of `require_alias` query string param for [create request](https://github.com/elastic/elasticsearch/blob/main/rest-api-spec/src/main/resources/rest-api-spec/api/create.json#L34-L77), while we do have one for [index request](https://github.com/elastic/elasticsearch/blob/main/rest-api-spec/src/main/resources/rest-api-spec/api/index.json#L101-L104). According to the documentation, they should have the same parameter set: https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-index_.html#docs-index-api-query-params.

@afharo, do you know if there's a centralized process for requesting updates to these REST specs, or is everyone expected to handle it separately?

Reply:
Those changes should be requested in http://github.com/elastic/elasticsearch-specification, which is not as up-to-date as we would like, but we're working on it. However, in this case, I opened https://github.com/elastic/elasticsearch-specification/pull/3954 to add it. I mentioned it above, but it's now hidden by GitHub.